<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outgoalsWindow</class>
 <widget class="QMainWindow" name="outgoalsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="accountListView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBudgetButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存本月预算</string>
    </property>
   </widget>
   <widget class="QChartView" name="budgetChartView" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="budgetComparisonLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;设置本月预算：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="amountbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupersonal_financial_manager">
    <property name="title">
     <string>personal financial manager 财务目标 消费计划</string>
    </property>
   </widget>
   <addaction name="menupersonal_financial_manager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
